libV2/utils/log.js

```diff
@@ -11,51 +11,67 @@ werelogs.configure(loggerConfig);
 const rootLogger = new werelogs.Logger('Utapi');
 
 class LoggerContext {
-    constructor(defaults) {
+    constructor(defaults, logger = null) {
         this._defaults = defaults;
+        this.logger = logger !== null ? logger : rootLogger;
```

Comment:
might do `logger || rootLogger`

Reply:
I like to be specific, I'm not testing for truthiness but if it's been set.